remix-lib/src/helpers/compilerHelper.js

```diff
@@ -13,7 +13,7 @@ function compilerInput (contracts) {
     settings: {
       optimizer: {
```

Comment:
Not, if you don't supply this entire block it will default to disabled and if you supply explicitly disabled, the `runs` field is not used anyway.